Bottom: 5e3762af012acd9974654627e89fad5393d43d62
Top:    03b18e8aab4a45da539dc081626c2faf86362a91
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 17:27:51 +0100

Refresh of hid-gtk-turn-on-defines-to-deb

---

diff --git a/src/hid/gtk/ghid-main-menu.c b/src/hid/gtk/ghid-main-menu.c
index 6c0ca01..4ae8c9a 100644
--- a/src/hid/gtk/ghid-main-menu.c
+++ b/src/hid/gtk/ghid-main-menu.c
@@ -347,7 +347,7 @@ ghid_main_menu_real_add_resource (GHidMainMenu *menu, GtkMenuShell *shell,
               int pos;
 
               children = gtk_container_get_children (GTK_CONTAINER (shell));
-              pos = g_list_length (shell->children);
+              pos = g_list_length (children);
               g_list_free (children);
 
               if (strcmp (res->v[i].value, "@layerview") == 0)
